.github/workflows/news-fragment.yml

```diff
@@ -38,6 +38,7 @@ jobs:
 
       - name: Check news fragment
         run: >
+          python -m pip install --upgrade uv &&
```

Comment:
This seems like an unrelated change?

Reply:
let me rebase it. It's merged in another PR.